examples/get-started/package.json

```diff
@@ -14,6 +14,6 @@
         "typescript": "^4.9.5"
     },
     "dependencies": {
-        "@mlc-ai/web-llm": "^0.2.6"
+        "@mlc-ai/web-llm": "file:../../dist"
```

Comment:
It is better to point npm version, as it is an example I expect to copy files and test it out. If we point local build it might cause confusion. wdyt?